test/environment/rlimit-bad.exp

```diff
@@ -10,7 +10,7 @@ match_max 100000
 send -- "firejail --rlimit-fsize=-1024\r"
 expect {
 	timeout {puts "TESTING ERROR 0\n";exit}
-	"invalid rlimit"
+	"invalid rlimit-fsize in profile file. Only use positive numbers and k, m or g suffix."
```

Comment:
This is a test for the command line option --rlimit-fsize, not for the option inside a profile file.
"invalid rlimit-fsize. Only use positive numbers and k, m or g suffix."